tests/robustness/multiclient_test/multiclient_test.go

```diff
@@ -150,6 +150,14 @@ func TestManySmallFilesAcrossDirecoryTree(t *testing.T) {
 
 	ctx := testlogging.ContextWithLevel(t, testlogging.LevelInfo)
 	th.RunN(ctx, t, numClients, f)
+
+	// Run gc action once
+	runGCFcn := func(ctx context.Context, t *testing.T) { //nolint:thelper
+		t.Log("run maintenance")
+		_, err := eng.ExecAction(ctx, engine.GCActionKey, nil)
+		require.NoError(t, err)
+	}
+	th.RunN(ctx, t, 1, runGCFcn)
```

Comment:
Is there a reason to use `th.RunN` with 1 client vs `th.Run`? I'm pretty rusty in this code path 😅

Reply:
Related question: why run maintenance here (in  `TestManySmallFilesAcrossDirecoryTree`)?

Reply:
I had two choices either create a new test or add it to one of the existing tests that perform other operations which impacts maintenance. I can add a new test if that is preferable.

Reply:
> Is there a reason to use `th.RunN` with 1 client vs `th.Run`? I'm pretty rusty in this code path 😅

Yes, I tried `th.Run`, but the test errored out. I can take that as an enhancement/refactor item for later.